src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudIT.java

```diff
@@ -21,7 +21,7 @@
 import com.cloudbees.plugins.credentials.SystemCredentialsProvider;
 import com.cloudbees.plugins.credentials.domains.Domain;
 import com.cloudbees.plugins.credentials.domains.DomainRequirement;
-import com.google.api.services.compute.model.AccessConfig;
+import com.google.api.services.compute.model.AccessConfig;	import com.google.api.services.compute.model.*;
```

Comment:
Imports should be on separate line. Fully qualified imports are preferred to wildcards.